app/controllers/api/v2/template_combinations_controller.rb

```diff
@@ -36,9 +62,13 @@ def destroy
       end
 
       def find_parent_config_template
```

Comment:
also unused?  I can't see any references to `@config_template`, I'm pretty sure it's all going through `nested_obj` now.